gpii/node_modules/settingsHandlers/src/settingsHandlerUtilities.js

```diff
@@ -298,6 +298,12 @@ gpii.settingsHandlers.writeFile = function (content, options) {
  */
 gpii.settingsHandlers.handleFileSolutionEntry = function (solutionEntry, modifier, parser, isWrite) {
     var options = solutionEntry.options;
+    var initialContent = options.initialContent;
+
+    var exists = fs.existsSync(options.filename);
+    if(!exists){
+        fs.writeFileSync(options.filename, initialContent, options.encoding || "utf-8");
+    }
 
```

Comment:
I can't comment much on this except to say 
1) is sync file creation OK
2) what about error handling
    * what should be the UK if file creation fails?
    * How is exception handling / error handling implemented. There doesn't seem to be any in this file.
3) Is nice that you have generalised the initialContent so can work with other solutions

@amb26 this is probably more your ball court